### PR TITLE
bug-1189121-remove-moz-expressions-from-bedrock

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

**Comment by TheJJ100100** [comment]
Thank You @alexgibson

**Comment by alexgibson** [comment]
Added a commit to tidy up a bit and removed some other vendor cruft. Thanks again @TheJJ100100!

**Comment by TheJJ100100** [comment]
Thanks @alexgibson

**Comment by schalkneethling** [comment]
Any reason this has not been merged yet? Is it the two commits?

**Comment by alexgibson** [comment]
@schalkneethling - just needs review. I'm preserving the two commits to @TheJJ100100 gets credit for the bug.

**Comment by TheJJ100100** [comment]
@alexgibson I was wondering too :)

On 26 August 2015 at 14:42, Alex Gibson <notifications@github.com> wrote:

> @schalkneethling <https://github.com/schalkneethling> - just needs
> review. I'm preserving the two commits to @TheJJ100100
> <https://github.com/TheJJ100100> gets credit for the bug.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/bedrock/pull/3240#issuecomment-135025250>.
>



-- 

<http://jab-jab.co.uk>

Joseph Bywater / A Student
jabjab.co.uk@gmail.com


http://jab-jab.co.uk

[image: Twitter] <https://twitter.com/thejj100100> [image: Youtube]
<https://www.youtube.com/user/TheJJ100100> [image: Instagram]
<http://www.instagram.com/kiwifruitapple/> [image: flickr]
<https://www.flickr.com/photos/129058144@N06/>

File: media/css/sandstone/sandstone-resp.less [hunk 1]
```diff
@@ -518,11 +518,7 @@ select.error {
     text-align: center;
     line-height: 16px;
     background-color: #aa1401;
-    background-image: -webkit-gradient(linear, left top, left bottom, from(rgba(218, 81, 50, 0.9)), to(rgba(169, 19, 0, 1)));
     background-image: -webkit-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
-    background-image:    -moz-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
-    background-image:     -ms-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
-    background-image:      -o-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
     background-image:         linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
```
Comment (schalkneethling) [inline media/css/sandstone/sandstone-resp.less]: indentation here needs a fix.

File: media/css/sandstone/video-resp.less [hunk 2]
```diff
@@ -43,10 +43,8 @@ a.mozilla-video-control-overlay {
     left: 0;
     width: 100%;
     background: #444;
-    background: -webkit-gradient(radial,center center,0,center center,100%,from(#fff), to(#000));
     background: -webkit-radial-gradient(center center, rgba(255,255,255,1) 0, rgba(0,0,0,0.9) 100%);
-    background: -moz-radial-gradient(center center, rgba(255,255,255,1) 0, rgba(0,0,0,0.9) 100%);
-    background: radial-gradient(center center, rgba(255,255,255,1) 0, rgba(0,0,0,0.9) 100%);
+    background:         radial-gradient(center center, rgba(255,255,255,1) 0, rgba(0,0,0,0.9) 100%);
```
Comment (schalkneethling) [inline media/css/sandstone/video-resp.less]: And here as well.

File: media/css/sandstone/video.less [hunk 3]
```diff
@@ -40,10 +40,8 @@ a.mozilla-video-control-overlay {
     left: 0;
     width: 100%;
     background: #444;
-    background: -webkit-gradient(radial,center center,0,center center,100%,from(#fff), to(#000));
     background: -webkit-radial-gradient(center center, rgba(255,255,255,1) 0, rgba(0,0,0,0.9) 100%);
-    background: -moz-radial-gradient(center center, rgba(255,255,255,1) 0, rgba(0,0,0,0.9) 100%);
-    background: radial-gradient(center center, rgba(255,255,255,1) 0, rgba(0,0,0,0.9) 100%);
+    background:         radial-gradient(center center, rgba(255,255,255,1) 0, rgba(0,0,0,0.9) 100%);
```
Comment (schalkneethling) [inline media/css/sandstone/video.less]: And here :)

**Comment by schalkneethling** [comment]
Hah, so we only have `-webkit-` that still needs a vendor prefix. Interesting. Code looks good, will pull it down and test.

File: media/css/sandstone/sandstone-resp.less (hunk 1 above)
Comment (alexgibson) [inline media/css/sandstone/sandstone-resp.less]: Actually this is intentional as per our [CSS styleguide formatting](http://mozweb.readthedocs.org/en/latest/reference/css-style.html#formatting-css)

File: media/css/sandstone/video-resp.less (hunk 2 above)
Comment (alexgibson) [inline media/css/sandstone/video-resp.less]: Same here.

File: media/css/sandstone/video.less (hunk 3 above)
Comment (alexgibson) [inline media/css/sandstone/video.less]: Same here :)

File: media/css/sandstone/sandstone-resp.less (hunk 1 above)
Comment (schalkneethling) [inline media/css/sandstone/sandstone-resp.less]: Oh, alrighty. Was not aware of that.

Comment (TheJJ100100) [inline media/css/sandstone/sandstone-resp.less]: @alexgibson @schalkneethling What happened? I did what I was told :P

On 26 August 2015 at 18:08, Schalk Neethling <notifications@github.com>
wrote:

> In media/css/sandstone/sandstone-resp.less
> <https://github.com/mozilla/bedrock/pull/3240#discussion_r38007479>:
>
> >      background-image: -webkit-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
> > -    background-image:    -moz-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
> > -    background-image:     -ms-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
> > -    background-image:      -o-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
> >      background-image:         linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
>
> Oh, alrighty. Was not aware of that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/bedrock/pull/3240/files#r38007479>.
>



-- 

<http://jab-jab.co.uk>

Joseph Bywater / A Student
jabjab.co.uk@gmail.com


http://jab-jab.co.uk

[image: Twitter] <https://twitter.com/thejj100100> [image: Youtube]
<https://www.youtube.com/user/TheJJ100100> [image: Instagram]
<http://www.instagram.com/kiwifruitapple/> [image: flickr]
<https://www.flickr.com/photos/129058144@N06/>

Comment (TheJJ100100) [inline media/css/sandstone/sandstone-resp.less]: @alexgibson @schalkneethling I think I did anyway!

On 26 August 2015 at 18:18, Joseph Bywater <jabjab.co.uk@gmail.com> wrote:

> @alexgibson @schalkneethling What happened? I did what I was told :P
>
> On 26 August 2015 at 18:08, Schalk Neethling <notifications@github.com>
> wrote:
>
>> In media/css/sandstone/sandstone-resp.less
>> <https://github.com/mozilla/bedrock/pull/3240#discussion_r38007479>:
>>
>> >      background-image: -webkit-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
>> > -    background-image:    -moz-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
>> > -    background-image:     -ms-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
>> > -    background-image:      -o-linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
>> >      background-image:         linear-gradient(top, rgba(218, 81, 50, 0.9), rgba(169, 19, 0, 1));
>>
>> Oh, alrighty. Was not aware of that.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mozilla/bedrock/pull/3240/files#r38007479>.
>>
>
>
>
> --
>
> <http://jab-jab.co.uk>
>
> Joseph Bywater / A Student
> jabjab.co.uk@gmail.com
>
>
> http://jab-jab.co.uk
>
> [image: Twitter] <https://twitter.com/thejj100100> [image: Youtube]
> <https://www.youtube.com/user/TheJJ100100> [image: Instagram]
> <http://www.instagram.com/kiwifruitapple/> [image: flickr]
> <https://www.flickr.com/photos/129058144@N06/>
>
>


-- 

<http://jab-jab.co.uk>

Joseph Bywater / A Student
jabjab.co.uk@gmail.com


http://jab-jab.co.uk

[image: Twitter] <https://twitter.com/thejj100100> [image: Youtube]
<https://www.youtube.com/user/TheJJ100100> [image: Instagram]
<http://www.instagram.com/kiwifruitapple/> [image: flickr]
<https://www.flickr.com/photos/129058144@N06/>

Comment (alexgibson) [inline media/css/sandstone/sandstone-resp.less]: @TheJJ100100 - you didn't do anything, not to worry. The code is fine :)

Comment (TheJJ100100) [inline media/css/sandstone/sandstone-resp.less]: Oh, okay, I just got an email and it looked confusing!

**Comment by schalkneethling** [comment]
Do we still actually *need* the `-webkit-` prefix? Seems like it supports the standard:
https://www.chromestatus.com/features/5785905063264256

http://codepen.io/chriscoyier/pen/zFoxn
http://codepen.io/chriscoyier/pen/iuaDL

**Comment by alexgibson** [comment]
There are lots of mobile WebKit-based browsers that can't be updated still in the wild, so I think this last vendor prefix will be around for some time yet unfortunately. This is consistent with the level of support we're currently providing across the rest of the site.

**Comment by schalkneethling** [comment]
Ah bummer, in that case r+

**Comment by alexgibson** [comment]
Thanks @schalkneethling! And thanks again for the PR @TheJJ100100 :cake:

**Comment by TheJJ100100** [comment]
@schalkneethling @alexgibson Thanks for all the support!  ;)

**Pull request merged**